piper/mousemap.py

```diff
@@ -119,8 +119,13 @@ def __init__(self, layer, ratbagd_device, spacing=10, *args, **kwargs):
         self._children = []
         self._highlight_element = None
 
-        self._handle = Rsvg.Handle.new_from_file(svg_path)
-        self._svg_data = etree.parse(svg_path)
+        uis = Gio.UnixInputStream.new(fd.fileno(), False)
+        self._handle = Rsvg.Handle.new_from_stream_sync(uis, None,
+                                                        Rsvg.HandleFlags.FLAGS_NONE,
+                                                        None)
+        # Same fd, so let's reset it before we parse it
+        fd.seek(0)
```

Comment:
Should we perhaps reset it after reading it in the `DeviceRow`, and again here? I don't know how costly this is, but it is more "future proof" to just reset it every time we read it.